masakari-controller/controller/masakari_worker.py

```diff
@@ -58,9 +64,16 @@ def __init__(self, config_object):
 
 #        self.WAIT_SYNC_TIME_SEC = 60
 
+        msg = "END __init__"
+        LOG.debug(self.rc_util.msg_with_thread_id(msg))
+
     def _get_vm_param(self, uuid):
```

Comment:
Should this kind of tiny method looks like utility methods have BEGIN/END debug log? It's just call rc_util_api.do_instance_show() and it also output the very similar log. I think it reduces log readability and traceability.